zerver/models.py

```diff
@@ -550,6 +555,16 @@ class PreregistrationUser(models.Model):
 
     realm = models.ForeignKey(Realm, null=True) # type: Optional[Realm]
 
+class OutgoingWebhookBot(UserProfile):
+    base_url = models.TextField() # type: text_type
+    service_api_key = models.TextField() # type: text_type
```

Comment:
@timabbott The service_api_key are provided by the external services that zulip communicates with. I thought it was best to make it a variable length field as the key length may vary across services